packages/xrpl/test/integration/transactions/credentialDelete.test.ts

```diff
@@ -0,0 +1,82 @@
+import { stringToHex } from '@xrplf/isomorphic/utils'
+import { assert } from 'chai'
+
+import { AccountObjectsResponse } from '../../../src'
+import { CredentialAccept } from '../../../src/models/transactions/CredentialAccept'
+import { CredentialCreate } from '../../../src/models/transactions/CredentialCreate'
+import { CredentialDelete } from '../../../src/models/transactions/CredentialDelete'
+import serverUrl from '../serverUrl'
+import {
+  setupClient,
+  teardownClient,
+  type XrplIntegrationTestContext,
+} from '../setup'
+import { generateFundedWallet, testTransaction } from '../utils'
+
+describe('CredentialDelete', function () {
+  // testContext wallet acts as issuer in this test
+  let testContext: XrplIntegrationTestContext
+
+  beforeAll(async () => {
+    testContext = await setupClient(serverUrl)
+  })
+  afterAll(async () => teardownClient(testContext))
+
+  it('base', async function () {
+    const subjectWallet = await generateFundedWallet(testContext.client)
+
+    const credentialCreateTx: CredentialCreate = {
+      TransactionType: 'CredentialCreate',
+      Account: testContext.wallet.classicAddress,
+      Subject: subjectWallet.classicAddress,
+      CredentialType: stringToHex('Test Credential Type'),
+    }
+
+    await testTransaction(
+      testContext.client,
+      credentialCreateTx,
+      testContext.wallet,
+    )
+
```

Comment:
This is already tested in CredentialCreate. The flow has to remain identical to test delete, wouldn't think we need to verify again since the other test will fail if that does not work. I think ideally all 3 credential tests could be combined with account object checks after each transaction, left them as separate for consistency with the repo though. Am open to combining if you'd agree, would be simpler

Reply:
fair enough, have added